docs/standard/serialization/system-text-json-converters-how-to.md

```diff
@@ -240,7 +241,7 @@ The JSON output from serialization looks like the following example:
 }
 ```
 
-The [unit tests folder](https://github.com/dotnet/corefx/blob/master/src/System.Text.Json/tests/Serialization/) in the `System.Text.Json.Serialization` namespace has more examples of custom converters that handle non-string-key dictionaries.
+The [unit tests folder](https://github.com/dotnet/corefx/blob/f77914d4c9045a01b3a91b63c28805f24850c274/src/System.Text.Json/tests/Serialization/) in the `System.Text.Json.Serialization` namespace has more examples of custom converters that handle non-string-key dictionaries.
 
 ### Support polymorphic deserialization
```

Comment:
This section needs to be corrected a bit now. Specifically, the following opening isn't accurate:

> Polymorphic serialization doesn't require a custom converter, but deserialization does require a custom converter.